features/nanostack/mbed-mesh-api/source/MeshInterfaceNanostack.cpp

```diff
@@ -130,6 +134,11 @@ nsapi_error_t Nanostack::Interface::register_phy()
 {
     NanostackLockGuard lock;
 
+    if (_device_id >= 0) {
```

Comment:
I have an equivalent but differently-written equivalent to this hunk in the pending https://github.com/ARMmbed/mbed-os/pull/7818/files. Could you make it match, to avoid conflicts?

Reply:
I removed my changes to rf phy registration from MeshInterfaceNanostack.cpp. Issues will be fixed in you pull request.